app/controllers/MaintenanceController.scala

```diff
@@ -18,41 +21,43 @@ class MaintenanceController @Inject()(sil: Silhouette[WkEnv],
     extends Controller
     with FoxImplicits {
 
+  private val maintenanceDuration: FiniteDuration = 10 minutes
+
   def info: Action[AnyContent] = sil.UserAwareAction.async { implicit request =>
     for {
       expirationTime <- maintenanceDAO.getExpirationTime
-      isMaintenance = expirationTime.getTime >= System.currentTimeMillis
+      isMaintenance = !expirationTime.isPast
```

Comment:
You could also add `isFuture`, but also fine as-is.

Reply:
I decided against isFuture for the moment. Might add it later.